### PR TITLE
DOCS-235 Add a tutorial for streaming SQL from Confluent Cloud

### PR DESCRIPTION
**Pull request by JakeSCahill**

### CODE REVIEW

File: tutorials/modules/ROOT/pages/create-materialized-view-from-kafka.adoc [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+'keyFormat'='bigint',
+'valueFormat'='json-flat');
+```
+
+. Submit a job to your cluster that will monitor your `trade` topic for changes and store them in a map. The processing guarantee tells Hazelcast to save the current offsets so that the cluster can resume the job even if the cluster restarts.
++
+```sql
+CREATE JOB ingest_trades
+OPTIONS (
+  'processingGuarantee' = 'exactlyOnce',
```
Comment (alaktionov) [inline tutorials/modules/ROOT/pages/create-materialized-view-from-kafka.adoc:256]: Redundant comma at the end of statement

File: tutorials/modules/ROOT/pages/create-materialized-view-from-kafka.adoc [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+```
++---------+---------+---------------------------------+------------+
+|       id|ticker   |   company                       |      amount|
++---------+---------+---------------------------------+------------+
+|        2|EORG     |Example Enterprise Organization  |          20|
+|        1|SORG     |Example Startup Organization     |          10|
++---------+---------+----------+----------------------+------------+
+```
++
+Leave this query running.
```
Comment (alaktionov) [inline tutorials/modules/ROOT/pages/create-materialized-view-from-kafka.adoc:300]: Hm, this query is not running, it was executed and finished. Maybe we can remove this line?

File: tutorials/modules/ROOT/pages/create-materialized-view-from-kafka.adoc [hunk 3]
```diff
@@ -0,0 +1,349 @@
...
+. In the SQL prompt, publish some more trades to the topic.
++
+[source,sql]
+----
+INSERT INTO trades VALUES
+  (3, 'SORG', 5.7, 23),
+  (4, 'EORG', 12, 54);
+----
++
+Your query in Management Center should now display the results, including your new trades. Your materialized view will continue to be updated for each new trade that's added to the topic in the Kafka cluster.
```
Comment (alaktionov) [inline tutorials/modules/ROOT/pages/create-materialized-view-from-kafka.adoc:311]: We need to re-execute step `SELECT * FROM trade_map;` to be able to see the updated results

File: tutorials/modules/ROOT/pages/create-materialized-view-from-kafka.adoc [hunk 4]
```diff
@@ -0,0 +1,349 @@
...
+
+. Click *Stop Query* in Management Center.
```
Comment (alaktionov) [inline tutorials/modules/ROOT/pages/create-materialized-view-from-kafka.adoc:313]: And again, the SELECT statement is run once and finished, no need to `Stop Query`

File: tutorials/modules/ROOT/pages/create-materialized-view-from-kafka.adoc [hunk 5]
```diff
@@ -0,0 +1,355 @@
...
+
+. Stop the streaming query by pressing kbd:[Ctrl + C] to close the connection to the SQL prompt.
+
+== Step 3. Enrich the Data in the Kafka Messages
+
+Kafka messages are often small and contain minimal data to reduce network latency. For example, the `trades` topic does not contain any information about the company that's associated with a given ticker. To get deeper insights from data in Kafka topics, you can join query results with contextual data.
+
+. Open the SQL browser in Management Center.
+
+. Create a mapping to a new map in Hazelcast, in which to store the company information that you can use to enrich results from the `trades` topic.
```
Comment (angelasimms) [inline tutorials/modules/ROOT/pages/create-materialized-view-from-kafka.adoc:189]: I'd split this down a bit more....something like:

The new map is for storing the company information that you'll use to enrich results from the `trades` topic.

File: tutorials/modules/ROOT/pages/create-materialized-view-from-kafka.adoc [hunk 6]
```diff
@@ -0,0 +1,355 @@
...
+|SORG        |Example Startup Organization   |10            |
+|EORG        |Example Enterprise Organization|20            |
+```
+
+. Click *Stop Query*.
+
+
+== Step 4. Create a Materialized View
+
+You can set up an automated job to continuously run the streaming query and cache the results in a Hazelcast map. This map is the materialized
```
Comment (angelasimms) [inline tutorials/modules/ROOT/pages/create-materialized-view-from-kafka.adoc:235]: Missing full stop at end of sentence.

File: tutorials/modules/ROOT/pages/create-materialized-view-from-kafka.adoc [hunk 7]
```diff
@@ -0,0 +1,355 @@
...
+  'client.dns.lookup'='use_all_dns_ips',
+  -- Best practice for higher availability in Apache Kafka clients prior to 3.0
+  'session.timeout.ms'='45000',
+  'auto.offset.reset'='earliest'
+);
+----
++
+TIP: You can find your previous mapping query in the *History* tab of the SQL browser.
+
+. In the SQL prompt, enter the same streaming query that gave no results last time.
```
[the remark this answers is not in the document]
Comment (JakeSCahill) [reply tutorials/modules/ROOT/pages/create-materialized-view-from-kafka.adoc]: We don't usually use 'at' when referring to command prompts.

**Review by alaktionov — APPROVED** [review]

**Review by angelasimms — APPROVED** [review]

**Pull request merged**